### PR TITLE
fix: add production target to api

### PR DESCRIPTION
**Pull request by Emnaghz**
# Motivation

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (Storybook)
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Emnaghz** [comment]
@yassinedorbozgithub ready to be tested! :rocket:

**Comment by yassinedorbozgithub** [comment]
> @yassinedorbozgithub ready to be tested! 🚀

@Emnaghz, review process is started :technologist:

**Review by yassinedorbozgithub — APPROVED** [review]

File: api/Dockerfile [hunk 1]
```diff
@@ -4,32 +4,46 @@ WORKDIR /app
...
-RUN npm i --verbose --maxsockets 6
 
-FROM node:18-alpine AS runner
+FROM node:18-alpine AS production
 
 WORKDIR /app
 
-COPY --from=installer /app/ .
+COPY --from=builder /app/package*.json ./
+COPY --from=builder /app/ .
```
Comment (marrouchi) [inline api/Dockerfile:39]: Copying the entire /app directory may include unnecessary files, increasing image size. Copy only the necessary build artifacts or directories needed for the application to run.

**Review by marrouchi — CHANGES_REQUESTED** [review]

**Comment by marrouchi** [comment]
Closed in favor of https://github.com/Hexastack/Hexabot/pull/98

**Pull request closed without merging**